doc/memos/rdm0.md

```diff
@@ -0,0 +1,153 @@
+
+- RDM: 0
+- Title: RIOT Developer Memo Format, Publishing and Maintenance Process
+- Author: Emmanuel Baccelli  
+- Date: October 2018
+
+## Abstract
+
+This memo describes a template for memos produced by the RIOT maintainers 
+community, targeting primarily RIOT developers.
+
+
+## Status
+
+This document is a product of the community of RIOT maintainers, and aims to 
+represent the consensus within this community. The content of this document is
+licensed with a Creative Commons CC-BY-SA license.
+
+## Terminology
+
+This memo uses the [RFC2119](https://www.ietf.org/rfc/rfc2119.txt) terminology and the following acronyms:
```

Comment:
I personally perfer the HTML versions of RFCs (and usually use it when I reference it within the codebase), but if you insist on TXT, maybe rather use the rfc-editor.org ones, since they are AFAIK the official ones.